packages/web/src/components/gcds-link/gcds-link.tsx

```diff
@@ -0,0 +1,221 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Watch,
+  Prop,
+  State,
+  h,
+} from '@stencil/core';
+import { assignLanguage, observerConfig } from '../../utils/utils';
+import { inheritAttributes } from '../../utils/utils';
+import i18n from './i18n/i18n';
+
+@Component({
+  tag: 'gcds-link',
+  styleUrl: 'gcds-link.css',
+  shadow: true,
+})
+export class GcdsLink {
+  @Element() el: HTMLElement;
+
+  private shadowElement?: HTMLElement;
+
+  /**
+   * Props
+   */
+
+  /**
+   * Set the link size
+   */
+  @Prop({ mutable: true }) size: 'regular' | 'small' | 'inherit' = 'regular';
+
+  @Watch('size')
+  validateSize(newValue: string) {
+    const values = ['regular', 'small', 'inherit'];
+
+    if (!values.includes(newValue)) {
+      this.size = 'regular';
+    }
+  }
+
+  @Watch('display')
+  validateDisplay(newValue: string) {
+    const values = ['block', 'inline', 'inline-block'];
+
+    if (!values.includes(newValue)) {
+      this.display = 'block';
+    }
+  }
+
+  /**
+   * Link props
+   */
+
+  /**
+   * Sets the display behavior of the link
+   */
+  @Prop() display: 'block' | 'inline' | 'inline-block' = 'inline';
+
+  /**
+   * The href attribute specifies the URL of the page the link goes to
+   */
+  @Prop() href!: string;
+
+  /**
+   * The rel attribute specifies the relationship between the current document and the linked document
+   */
+  @Prop() rel: string | undefined;
+
+  /**
+   * The target attribute specifies where to open the linked document
+   */
+  @Prop() target?: string = '_self';
+
+  /**
+   * Whether the link is external or not
+   */
+  @Prop() external: false;
```

Comment:
Can you set the type to boolean and then the default value to false? And can we also flag it as optional prop?